### PR TITLE
Start migrating to gradle (non-working solution)

### PR DESCRIPTION
**Pull request by SandwichBtw**
I will fix it tomorrow but here is the start...

### CODE REVIEW

**Comment by natereprogle** [comment]
Thanks for your help! I see this is still WIP, so I converted it to a draft. I’ll continue looking at this, please add commits if you do more work on this!

**Comment by SandwichBtw** [comment]
Ofcourse! I'll try to make another commit tomorrow.

**Comment by natereprogle** [comment]
@SandwichBtw Please review the changes I made in the SandwichBtw-master branch to the gradle files, and rebase them into your PR. I unfortunately cannot push any changes into this fork since it was based on the master branch of the project, and that branch is protected for a reason.

File: honeypot-core/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    implementation(libs.boosted.yaml)
+    implementation(libs.bstats)
+
+    compileOnly(libs.worldguard)
+    compileOnly(libs.griefprevention)
+    compileOnly(libs.lands)
+}
+
+tasks.shadowJar {
+    archiveFileName = "honeypot-core${version}.jar"
```
[the remark this answers is not in the document]
Comment (natereprogle) [reply honeypot-core/build.gradle.kts:37]: Fixed

**Review by natereprogle — APPROVED** [review]

**Review by natereprogle — APPROVED** [review]
Looks good!

**Comment by natereprogle** [comment]
TODO:
* Fix publishing since that's broken now
* Update Java CI with Maven since it's no longer maven lol

**Pull request merged**